internal/commands/scan.go

```diff
@@ -502,7 +509,18 @@ func scanCreateSubCommand(
 		"",
 		fmt.Sprintf("Parameters to use in SCA resolver (requires --%s).", commonParams.ScaResolverFlag),
 	)
-	createScanCmd.PersistentFlags().String(commonParams.ScanTypes, "", "Scan types, ex: (sast,iac-security,sca,api-security)")
+	if len(wrappers.FeatureFlags) == 0 {
```

Comment:
why do you need this condition? 
if FF is not received you have to know it before (when you call the FF service) and fail the command if FF is necessary
but maybe you have to ignore containers when FF was not received